docs/includes/virtualenv.txt

```diff
@@ -0,0 +1,5 @@
+.. note:: You can install, upgrade, and remove Python packages using ``pip``, the Python package installer. By default, ``pip`` will attempt to install packages system-wide. Use ``venv`` or ``virtualenv`` to avoid installing Python packages globally and to isolate the installation of dependencies of a specific application.
+          
+          Run the command ``python3 -m venv .venv`` to create and manage your virtual environment. To begin using the virtual environment, use the command ``source .venv/bin/activate`` to activate.   
+        
+          You can also use `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/install.html>`_ which provides the ``mkvirtualenv`` shell function and is a set of extensions to the ``virtualenv`` tool. The extensions include wrappers for creating and deleting virtual environments and otherwise managing your development workflow. 
```

Comment:
Great use of an include! Was tempted to close out the long-standing https://github.com/freedomofpress/securedrop/issues/1319 but there are a few pointers in there that may still be worth following up on.